nimare/stats.py

```diff
@@ -147,26 +153,26 @@ def null_to_p(test_value, null_array, tail="two"):
     else:
         reconstruct = False
 
-    # TODO: this runs in N^2 time; is there a more efficient alternative?
-    p = np.array([stats.percentileofscore(null_array, v, "strict") for v in test_value])
-    p /= 100.0
     if tail == "two":
-        p = (0.5 - np.abs(p - 0.5)) * 2
-    elif tail == "upper":
-        p = 1 - p
-    elif tail != "lower":
-        raise ValueError('Argument "tail" must be one of ["two", "upper", "lower"]')
+        test_value = np.abs(test_value)
+        null_array = np.abs(null_array)
```

Comment:
Does this mean that `tail = "two"` will be the same as `tail = "upper"` when the null array and test values are all positive?

Reply:
Oh, good point—this will break for asymmetric (or non-centered) distributions. I was focusing on the ALE subtraction case, where the null distribution will be symmetric, but this won't be true for a standard ALE analysis.

The downside of reverting to our previous approach (double the minimum of the smaller one-tailed p-value) is that it's conservative in the case of a symmetric distribution, plus it requires doing two one-sided tests. (We weren't actually doing two one-tailed tests before, which was a bug, though in practice the difference would be negligible in nearly all cases.)

My proposal in light of the above is to revert to the previous procedure as the default for `tail='two'`, but also add a `symmetric=False` argument that, when set to `True`, will use the above approach (taking the absolute value), which is a little more powerful. Then we can use the latter whenever we know the null distribution is symmetric (e.g., for subtractions). Sound good?

Reply:
That sounds good to me. One (naive) question though- could we just center the array and adjust the test value accordingly, then just use your approach?

Something like:
```python
if tail == "two":
    # mean center
    m = null_array.mean()
    test_value -= m
    null_array -= m
    # take absolute values
    test_value = np.abs(test_value)
    null_array = np.abs(null_array)
```

Reply:
> could we just center the array and adjust the test value accordingly, then just use your approach?

If we knew for a fact that the distribution was symmetric, I think we could do that. But we don't know that, and it seems very unlikely to be true for, e.g., ALE statistics, so I think we should probably keep it this way.

Reply:
Okay sounds good then. Thanks!